tests/test_snapshot_download.py

```diff
@@ -3,12 +3,14 @@
 import time
 import unittest
 
+import requests
 from huggingface_hub import HfApi, Repository
 from huggingface_hub.snapshot_download import snapshot_download
 from tests.testing_constants import ENDPOINT_STAGING, PASS, USER
 
 
-REPO_NAME = "dummy-hf-hub-{}".format(int(time.time() * 10e3))
+REPO_NAME_PUBLIC = "dummy-hf-hub-{}".format(int(time.time() * 10e3))
+REPO_NAME_PRIVATE = "dummy-hf-hub-{}".format(int(time.time() * 10e3))
 
 
 class SnapshotDownloadTests(unittest.TestCase):
```

Comment:
For some reason I am not able to run these tests locally due to a permissions error:

```
src/huggingface_hub/repository.py:322: in __init__
    self.clone_from(repo_url=clone_from)
src/huggingface_hub/repository.py:432: in clone_from
    whoami_info = api.whoami(token)
src/huggingface_hub/hf_api.py:330: in whoami
    r.raise_for_status()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <Response [401]>

    def raise_for_status(self):
        """Raises :class:`HTTPError`, if one occurred."""
    
        http_error_msg = ''
        if isinstance(self.reason, bytes):
            # We attempt to decode utf-8 first because some servers
            # choose to localize their reason strings. If the string
            # isn't utf-8, we fall back to iso-8859-1 for all other
            # encodings. (See PR #3538)
            try:
                reason = self.reason.decode('utf-8')
            except UnicodeDecodeError:
                reason = self.reason.decode('iso-8859-1')
        else:
            reason = self.reason
    
        if 400 <= self.status_code < 500:
            http_error_msg = u'%s Client Error: %s for url: %s' % (self.status_code, reason, self.url)
    
        elif 500 <= self.status_code < 600:
            http_error_msg = u'%s Server Error: %s for url: %s' % (self.status_code, reason, self.url)
    
        if http_error_msg:
>           raise HTTPError(http_error_msg, response=self)
E           requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://huggingface.co/api/whoami-v2
```

Is there some environment variables that need to be set when testing locally?

Reply:
Yes, I think you need to set HUGGINGFACE_CO_STAGING = yes

Reply:
Note you should change it back since it will impact `huggingface_hub` in general

Reply:
Thanks for the tip!